CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.11.8]
```

Comment:
update the version in package.json?

Reply:
Sorry, the [README](https://github.com/AccelByte/justice-ui-library/blob/master/README.md#how-to-commit) doesn't mention updating `package.json`. I'll go ahead and update it 🙇‍♂️

Reply:
updated the version in package.json